### PR TITLE
[stable/odoo] Fix Odoo ingress to support multiple paths

### PR DESCRIPTION
**Pull request by miguelaeh**
Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/odoo/templates/ingress.yaml [hunk 1]
```diff
@@ -18,19 +17,20 @@ metadata:
...
             servicePort: http
-{{- if .tls }}
+  {{- end }}
+{{- if .Values.ingress.tls }}
   tls:
   - hosts:
     - {{ .name }}
     secretName: {{ .tlsSecret }}
 {{- end }}
 ---
```
Comment (alemorcuq) [inline stable/odoo/templates/ingress.yaml]: `---` is not needed anymore. Otherwise, lgtm.

**Review by alemorcuq — APPROVED** [review]

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Actually, I have a couple of questions:

1. What's wrong with the current approach?
2. Looking at the [values.yaml](https://github.com/helm/charts/blob/master/stable/odoo/values.yaml) there's no such thing as `.Values.ingress.tls` but there's a `tls` setting for each host.

**Comment by miguelaeh** [comment]
> What's wrong with the current approach?
https://github.com/bitnami/bitnami-docker-odoo/issues/74

> Looking at the values.yaml there's no such thing as .Values.ingress.tls but there's a tls setting for each host.
https://github.com/helm/charts/blob/941311b8643aeb5db4bbdb53296eb90c5c2b9903/stable/odoo/values.yaml#L151

**Comment by alemorcuq** [comment]
> > Looking at the values.yaml there's no such thing as .Values.ingress.tls but there's a tls setting for each host.
> 
> https://github.com/helm/charts/blob/941311b8643aeb5db4bbdb53296eb90c5c2b9903/stable/odoo/values.yaml#L151

```yaml
ingress:
  ## Set to true to enable ingress record generation
  enabled: false

  ## The list of hostnames to be covered with this ingress record.
  ## Most likely this will be just one host, but in the event more hosts are needed, this is an array
  hosts:
  - name: odoo.local
    path: /

    ## Set this to true in order to enable TLS on the ingress record
    ## A side effect of this will be that the backend odoo service will be connected at port 443
    tls: false
```

`tls` is defined for host, there's not a unique `.Values.ingress.tls`

**Comment by miguelaeh** [comment]
HI @alemorcuq ,
You were right, I have adapted it to support the TLS for each host.

**Pull request merged**